services/database/program.ts

```diff
@@ -23,4 +24,44 @@ async function getPrograms(): Promise<Program[]> {
     return programs;
 }
 
-export { getProgram, getPrograms };
+/**
+ * NOTE: https://www.prisma.io/docs/concepts/components/prisma-client/advanced-type-safety/operating-against-partial-structures-of-model-types
+ * getProgram takes the id parameter and returns the program associated with the programId
+ * @param newProgramData - data corresponding to new program
+ * @returns
+ */
+async function createProgram(
+    newProgramData: createProgramInput,
+): Promise<Program> {
+    const program = await prisma.program.create({
+        data: {
+            price: newProgramData.price,
+            start_date: newProgramData.startDate,
+            end_date: newProgramData.endDate,
+            weekday: newProgramData.weekday,
+            start_time_minutes: newProgramData.startTimeMinutes,
+            duration_minutes: newProgramData.durationMinutes,
+            space_total: newProgramData.spaceTotal,
+            space_available: newProgramData.spaceAvailable,
+            volunteer_space_total: newProgramData.volunteerSpaceAvailable,
+            volunteer_space_available: newProgramData.volunteerSpaceAvailable,
+            is_archived: newProgramData.isArchived,
+        },
+    });
+    return program;
+}
+
+/**
+ * deleteProgram takes the id parameter and deletes the program associated with the programId
+ * @param {string} id - programId
+ */
+async function deleteProgram(id: string): Promise<Program> {
```

Comment:
@feng-eric - same question about hard vs. soft deletes here

Reply:
Let's do hard delete for now and get the PR in. If we need soft delete later, we can make that change. Let's discuss with the entire team if there's a necessary use case for soft delete.